### PR TITLE
fix excel import after security patch

### PR DESCRIPTION
**Pull request by vishalya**

### CODE REVIEW

File: services/importer/lib/importer/excel2csv.rb [hunk 1]
```diff
@@ -64,14 +64,24 @@ def converted_filepath
...
-        [CartoDB.python_bin_path, newline_remover_path]
+      def newline_remover_command
+        [newline_remover_path]
+      end
+
+      def in2csv_command
+          "in2csv"
       end
 
+
```
Comment (gfiorav) [inline services/importer/lib/importer/excel2csv.rb]: Remove this extra line if you have the chance please!

File: services/importer/lib/importer/excel2csv.rb [hunk 2]
```diff
@@ -64,14 +64,24 @@ def converted_filepath
...
       def in2csv_warning_filter
         IN2CSV_WARNINGS.map { |w| ['grep', '-v', w.gsub('*', "\\*")] }
       end
 
-      def newline_remover_path
-        [CartoDB.python_bin_path, newline_remover_path]
+      def newline_remover_command
+        [newline_remover_path]
+      end
```
Comment (gfiorav) [inline services/importer/lib/importer/excel2csv.rb:77]: Did `CartoDB.python_bin_path` raise an error for you?

File: services/importer/lib/importer/excel2csv.rb [hunk 3]
```diff
@@ -64,14 +64,24 @@ def converted_filepath
 
       protected
 
+      def newline_remover_path
+        File.expand_path(NEWLINE_REMOVER_RELPATH, __FILE__)
+      end
+
+
```
Comment (gfiorav) [inline services/importer/lib/importer/excel2csv.rb]: Also remove this extra line too please!

**Review by gfiorav — COMMENTED** [review]
Made some comments, take a look!

File: services/importer/lib/importer/excel2csv.rb (hunk 2 above)
Comment (vishalya) [reply services/importer/lib/importer/excel2csv.rb:77]: yes, I thought it's not there for our code base

File: services/importer/lib/importer/excel2csv.rb (hunk 3 above)
Comment (vishalya) [reply services/importer/lib/importer/excel2csv.rb]: Blank line?

File: services/importer/lib/importer/excel2csv.rb (hunk 1 above)
Comment (vishalya) [reply services/importer/lib/importer/excel2csv.rb]: Blank line? Looks like hound didn't like it. will remove it.

**Review by gfiorav — APPROVED** [review]
Looks good to me!

📦 it!

**Pull request merged**